govcd/system_test.go

```diff
@@ -197,3 +197,16 @@ var INVALID_NAME = `*******************************************INVALID
 func init() {
 	testingTags["system"] = "system_test.go"
 }
+
+func (vcd *TestVCD) Test_QueryOrgVdcNetworkByName(check *C) {
```

Comment:
Should this function check these specifically new parameters which aren't available in different places? (as per discussion on function definition)

Reply:
Improved